sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/LogsHelperTests.cs

```diff
@@ -183,6 +183,36 @@ public void PropertiesContainEventIdAndEventNameIfSetOnLog()
             Assert.Equal(2, properties.Count);
         }
 
+        [Fact]
+        public void LogRecordAndAttributesContainEventIdAndEventName()
+        {
+            var logRecords = new List<LogRecord>();
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.AddInMemoryExporter(logRecords);
+                });
+                builder.AddFilter(typeof(LogsHelperTests).FullName, LogLevel.Trace);
+            });
+
+            var logger = loggerFactory.CreateLogger<LogsHelperTests>();
+
+            EventId id = new EventId(1, "TestEvent");
+
+            string log = "Log Information {EventId} {EventName}.";
+            logger.LogInformation(id, log, 100, "TestAttributeEventName");
```

Comment:
nit: can you please use named properties here?

Reply:
This is the pattern used in all other tests in this file. If there is no strong objection will leave it unmodified.